modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/BaseExportImportTestCase.java

```diff
@@ -52,17 +52,33 @@ public void testExportImport(
 			String externalReferenceCode, String name)
 		throws Exception {
 
+		Class<?> clazz = getClazz();
+
+		String actualJsonString = StringUtil.read(
+			clazz.getResourceAsStream("dependencies/" + actualFileName));
+
+		String expectedJsonString = StringUtil.read(
+			clazz.getResourceAsStream("dependencies/" + expectedFileName));
+
+		testExportImport(
+			actualFileName, actualJsonString, expectedJsonString,
+			externalReferenceCode, name);
+	}
+
+	public void testExportImport(
+			String actualFileName, String actualJSONString,
```

Comment:
@evibudai  i think you missed this one

Reply:
we need that there I think. We have to give an actualFileName explicitly when we're calling my test case.

Reply:
Ok. Maybe when you add the test case explicitly makes more sense. Since right now we only have a case with "actualFileName" it makes no sense to keep it like that